lms/djangoapps/course_api/blocks/views.py

```diff
@@ -337,6 +339,65 @@ def list(self, request, hide_access_denials=False):  # pylint: disable=arguments
         return response
 
 
+@method_decorator(transaction.non_atomic_requests, name='dispatch')
+@view_auth_classes(is_authenticated=False)
+class BlockMetadataView(DeveloperErrorViewMixin, ListAPIView):
+    """
+    **Use Case**
+
+        Returns the block metadata. Large data like index_dictionary related to
+        a block should be fetched using this API.
```

Comment:
Maybe explicitly add: ", because they are too large for the cache used by the course blocks/transformers API."